### ISSUE TITLE
Broken module

### ISSUE THREAD

**Issue by PrimozGodec** [conda-forge/fasttext-feedstock#9]
- [X] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:

It seems this module is somehow broken. I installed it with `conda install fasttext -c conda-forge` in my Python3.8 Conda environment. It has installed successfully but I cannot import it. 
```
>>> import fasttext
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'fasttext'
```
It is also not present in site-packages. I do not have an idea of what is happening. I tried that on macOS but also do not work on linux.

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
ca-certificates           2020.12.5            h033912b_0    conda-forge
certifi                   2020.12.5        py38h50d1736_1    conda-forge
fasttext                  0.9.2                h879752b_1    conda-forge
libcxx                    11.0.1               habf9029_0    conda-forge
libffi                    3.3                  h046ec9c_2    conda-forge
ncurses                   6.2                  h2e338ed_4    conda-forge
openssl                   1.1.1i               h35c211d_0    conda-forge
pip                       20.3.3             pyhd8ed1ab_0    conda-forge
python                    3.8.6           h624753d_4_cpython    conda-forge
python_abi                3.8                      1_cp38    conda-forge
readline                  8.0                  h0678c8f_2    conda-forge
setuptools                49.6.0           py38h50d1736_3    conda-forge
sqlite                    3.34.0               h17101e1_0    conda-forge
tk                        8.6.10               h0419947_1    conda-forge
wheel                     0.36.2             pyhd3deb0d_0    conda-forge
xz                        5.2.5                haf1e3a3_1    conda-forge
zlib                      1.2.11            h7795811_1010    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
     active environment : temp
    active env location : /Users/primoz/miniconda3/envs/temp
            shell level : 2
       user config file : /Users/primoz/.condarc
 populated config files : /Users/primoz/.condarc
          conda version : 4.9.0
    conda-build version : not installed
         python version : 3.7.4.final.0
       virtual packages : __osx=10.15.7=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /Users/primoz/miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /Users/primoz/miniconda3/pkgs
                          /Users/primoz/.conda/pkgs
       envs directories : /Users/primoz/miniconda3/envs
                          /Users/primoz/.conda/envs
               platform : osx-64
             user-agent : conda/4.9.0 requests/2.24.0 CPython/3.7.4 Darwin/19.6.0 OSX/10.15.7
                UID:GID : 501:20
             netrc file : None
           offline mode : False
```
</details>

**Comment by kylebgorman** [comment]
This doesn't install the Python extension module. It installs the command-line tool. So this is WAI. You should be able to install the module from PyPi (i.e., via pip) without too much effort.

**Comment by kylebgorman** [comment]
Closed via #12, we think
